gaas-ui/src/index.tsx

```diff
@@ -1,27 +1,29 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import App from "./components/App";
-import * as serviceWorker from "./serviceWorker";
-import { BrowserRouter } from "react-router-dom";
-import { CssBaseline } from "@material-ui/core";
-import { StylesProvider } from "@material-ui/styles";
-import GlobalStyles from "../src/components/GlobalStyles";
-import "./styles/_App.scss";
+import React from "react"
```

Comment:
Is there a reason all these semi-colons were taken away?

Reply:
Not sure how that happened. They have been added back in sorry.